README.md

```diff
@@ -39,8 +39,7 @@ SQLite and PostgreSQL are supported, with SQLite being the default for developme
 Use PostgreSQL in production for improved performance by adding the following settings:
 ```
 # Example PostgreSQL connection settings (default port is 5432)
-:database_backend: postgresql
-:postgresql_connection_string: postgres://foreman-proxy:changeme@localhost:5432/container_gateway
+:connection_string: postgres://foreman-proxy:changeme@localhost:5432/container_gateway
```

Comment:
I just verified that the following works:

```
:postgresql_connection_string: postgres://foreman-proxy:password/container_gateway
```

So I suppose we can remove the localhost designation along with the port since most users will use the local postgres.